.github/actions/setup/action.yml

```diff
@@ -4,13 +4,14 @@ runs:
   steps:
     - uses: actions/setup-node@v3
       with:
-        node-version-file: ".nvmrc"
+        node-version-file: '.nvmrc'
         registry-url: https://registry.npmjs.org
         cache: yarn
 
     # For provenance https://docs.npmjs.com/generating-provenance-statements#prerequisites
     - name: Install npm 9.5
       run: npm install -g npm@^9.5.0
+      shell: bash
```

Comment:
CI is broken missing shell: https://github.com/base-org/build-onchain-apps/actions/runs/6873486350